blingfire/src/lib.rs

```diff
@@ -76,7 +76,7 @@ where
             // Although the C++ function allocated an internal buffer with the parsed text, that's
             // not exposed. We'll have to to reserve `length` additional bytes in `destination` (as
             // `destination.len() == 0`) and parse the `source` string again.
-            destination.reserve(length as usize);
```

Comment:
Why `reserve_exact`? Happy to allocate more mem and as the docs says only use `reserve_exact` if you know better than the allocator

Reply:
I was thinking we know exactly how much we need in this case---but I could swap back.

Reply:
Meh, I thought about it and chose `reserve` based on doc :confused:, I changed to `reserve_exact` as I already merged those commits, tis ok